### PR TITLE
Make adjustments to Feature PR templates

### LINKED ISSUE #261: 🚧Make adjustments to Feature PR templates
### I have done the items below . . .

- [X] I have updated the title by replacing the '**_\<title\>_**' section.

### Description

Make various adjustments to the **Feature** and **Preview Feature** PR templates to cleanup the process and the PR card itself.

**The file names are:**
1. `feature-pr.md`
2. `preview-feature-pr.md`

### Acceptance Criteria

**This issue is finished when:**
- [x] Feature PR template updated
- [x] Preview Feature PR template updated

### ToDo Items

- [x] Draft pull request created and linked to this issue
- [X] Priority label added to issue (**_low priority_**, **_medium priority_**, or **_high priority_**)
- [X] Issue linked to the proper project
- [X] Issue linked to proper milestone

### Issue Dependencies

- #219 

### Related Work

- KinsonDigital/issues-repo#9

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by CalvinWilkinson**
<h1 style="font-weight:bold" align="center">Preview Feature Pull Request</h1>

<h2 style="font-weight:bold" align="center">✅Development Checklist✅</h2>

<details open><summary>🐛Bugs🐛</summary>

Contains Bug Fix(es)
- [ ] Yes
  - [ ] A bug-label label has been added to the pull request.
- [x] No
</details>

<details open><summary>🧨Breaking Change(s)🧨</summary>

Contains [breaking change(s)](https://docs.microsoft.com/en-us/dotnet/core/compatibility/#modifications-to-the-public-contract)
- [ ] Yes
  - [ ] A breaking-change-label label has been added to the pull request.
- [x] No
</details>

<details open><summary>✨Enhancements✨</summary>

Contains enhancement(s) that add a feature or behavior.
- [ ] Yes
  - [ ] An enhancement-label label has been added to the pull request.
- [x]  No
</details>

<details open><summary>⚙️Workflow (CI/CD) Changes⚙️</summary>

Contains changes to workflow files. These changes can only be done by the project maintainers.
- [ ] Yes
  - [ ] A workflow-label label has been added to the pull request.
- [x] No
</details>

<details open><summary>📃Documentation Updates📃</summary>

Contains changes that require updates to existing **_code_** and/or **_Velaptor product_** documentation.
- [x] Yes
  - [x] I have updated the documentation accordingly.
    - Choose the type of documentation.
    - [ ] Code Documentation
      - [ ] A documentation-code-label label has been added to the pull request.
    - [x] Product Documentation
      - [x] A documentation-product-label label has been added to the pull request.
- [ ] No
</details>

<details open><summary>🧪Unit Testing🧪</summary>

My change requires unit tests to be written.
- [ ] Yes
- [x] No
</details>

<details open><summary>🧪Manual Testing🧪</summary>

I have manually tested my changes using the project named **_VelaptorTesting_**.
- [ ] Yes
- [x] No
</details>

<h2 style="font-weight:bold" align="center">✅Code Review Checklist✅</h2>

- [x] Pull request title matches the title of the linked issue.
- [x] Associated issue exists and is linked to this pull request.
  - One issue per pull request.
- [x] My code follows the coding style of this project.
  - The style is enforced by the *.editorconfig* files in the project and displayed as warnings.  If there is an edge case with a coding style that should be ignored or changed, reach out, and let's discuss it.
- [x] A preview-label label has been added to the pull request.
- [x] All tests passed locally.
  - This is required because unit tests might pass locally but not in the CI environment during the status check process or vise-versa.
  - Tests might pass on the developer's machine but not necessarily on the code reviewer's machine.
  - This does not mean that the tests pass in both the local and CI environment.
  - Status checks run for this pull request when merging into the **_preview/v\*.\*.\*-preview.\*_** branch.  These status checks run every time a change is pushed to the pull request.  These checks validate version syntax, tagging, builds, unit tests, and more.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/feature-pr.md [hunk 1]
```diff
@@ -1,102 +1,35 @@
...
-- [ ] Yes
-- [ ] No
-</details>
+- [ ] Pull request title matches the title of the associated issue.
 
+- [ ] Associated issue exists and is linked to this pull request.
+  - One issue per pull request.
 
-<h2 style="font-weight:bold" align="center">✅Code Review Checklist✅</h2>
+- [ ] The labels attached to the PR match the labels attached to the associated issue.
```
Comment (KinsonDigitalAdmin) [inline .github/PULL_REQUEST_TEMPLATE/feature-pr.md:26]: Refactor to the value below:

```suggestion
- [ ] The labels attached to this PR match the labels attached to the associated issue.
```

**Review by KinsonDigitalAdmin — CHANGES_REQUESTED** [review]

**Review by KinsonDigitalAdmin — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE/feature-pr.md [hunk 2]
```diff
@@ -1,102 +1,35 @@
...
-  - [ ] I have updated the documentation accordingly.
-    - Choose the type of documentation.
-    - [ ] Code Documentation
-      - [ ] A ![documentation-code-label](https://user-images.githubusercontent.com/85414302/154672489-8079ed03-b8ff-41ff-9864-1e2ae55300cc.png) label has been added to the pull request.
-    - [ ] Product Documentation
-      - [ ] A ![documentation-product-label](https://user-images.githubusercontent.com/85414302/154672508-5ac50eb7-67f8-4cdf-92b9-fe5fcbd93b14.png) label has been added to the pull request.
-- [ ] No
-</details>
-
+- [ ] The **_head(source)_** branch for this pull request is a **_feature_** branch with the correct naming syntax in the following example.
```
Comment (kselena) [inline .github/PULL_REQUEST_TEMPLATE/feature-pr.md:4]: Refactor using info below:
```suggestion
- [ ] The **_head(source)_** branch for this pull request is a **_feature_** branch, with the correct naming syntax, in the following example:
```

File: .github/PULL_REQUEST_TEMPLATE/feature-pr.md [hunk 3]
```diff
@@ -1,102 +1,35 @@
...
+      ```
+    </details>
 
-My change requires unit tests.
-- [ ] Yes
-- [ ] No
-</details>
+- [ ] The **_head(source)_** branch for this pull request is created from a **_development_** branch with the name **_develop_**.
 
+💡For more info on branching, refer to the projects [branching documentation](../../Documentation/Branching.md).
```
Comment (kselena) [inline .github/PULL_REQUEST_TEMPLATE/feature-pr.md:16]: Use full word "information," not info.
```suggestion
💡For more information on branching, refer to the projects [branching documentation](../../Documentation/Branching.md).
```

File: .github/PULL_REQUEST_TEMPLATE/feature-pr.md [hunk 4]
```diff
@@ -1,102 +1,35 @@
...
 
-<details open><summary>🧪Manual Testing🧪</summary>
+- [ ] The **_base(target)_** branch for this pull request is a **_development_** branch with the name **_develop_**.
+  - A **_base(target)_** branch is the branch that is a source branch being merged into it.
```
Comment (kselena) [inline .github/PULL_REQUEST_TEMPLATE/feature-pr.md:19]: Refactor to same as Preview PR

File: .github/PULL_REQUEST_TEMPLATE/feature-pr.md [hunk 5]
```diff
@@ -1,102 +1,35 @@
...
-<h2 style="font-weight:bold" align="center">✅Code Review Checklist✅</h2>
+- [ ] The labels attached to this PR match the labels attached to the associated issue.
 
-<!-- Go over all of the following points, and put an `x` in all the boxes that apply. -->
-<!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
-- [ ] Pull request title matches the example below with the correct version.
-  - **Example:** 🚀Release To Preview - v1.2.3-preview.4
-- [ ] Issues exist and are linked to this pull request.
 - [ ] My code follows the coding style of this project.
   - The style is enforced by the *.editorconfig* files in the project and displayed as warnings.  If there is an edge case with a coding style that should be ignored or changed, reach out, and let's discuss it.
```
Comment (kselena) [inline .github/PULL_REQUEST_TEMPLATE/feature-pr.md:29]: Refactor everything to remove comma

```suggestion
  - The style is enforced by the *.editorconfig* files in the project and displayed as warnings.  If there is an edge case with a coding style that should be ignored or changed, reach out and let's discuss it.
```

File: .github/PULL_REQUEST_TEMPLATE/feature-pr.md [hunk 6]
```diff
@@ -1,102 +1,35 @@
...
+
+- [ ] I have manually tested my code changes to the best of my ability.
+
 - [ ] All tests passed locally.
   - This is required because unit tests might pass locally but not in the CI environment during the status check process or vise-versa.
```
Comment (kselena) [inline .github/PULL_REQUEST_TEMPLATE/feature-pr.md:34]: Misspelling of vice
```suggestion
  - This is required because unit tests might pass locally but not in the CI environment during the status check process or vice-versa.
```

File: .github/PULL_REQUEST_TEMPLATE/preview-feature-pr.md [hunk 7]
```diff
@@ -1,114 +1,56 @@
 <!--
-    !! NOTE !! - ANY CONTRIBUTOR CAN CREATE PREVIEW FEATURE PULL REQUESTS
+    !! NOTE !! - ONLY PROJECT OWNERS AND MAINTAINERS CAN CREATE PRODUCTION AND PREVIEW RELEASE PULL REQUESTS
     Please use the "preview-feature-pr" pull request template if you have contributions to make.
 -->
 <!--suppress HtmlDeprecatedAttribute -->
 <h1 style="font-weight:bold" align="center">Preview Feature Pull Request</h1>
+<h2 style="font-weight:bold" align="center">✅Code Review Checklist✅</h2>
 
+- [ ] The **_head(source)_** branch for this pull request is a **_preview feature_** branch with the correct naming syntax in the following example.
```
Comment (kselena) [inline .github/PULL_REQUEST_TEMPLATE/preview-feature-pr.md:9]: Use colon, not period at the end and use commas in the middle of the sentence.
```suggestion
- [ ] The **_head(source)_** branch for this pull request is a **_preview feature_** branch, with the correct naming syntax, in the following example:
```

File: .github/PULL_REQUEST_TEMPLATE/preview-feature-pr.md [hunk 8]
```diff
@@ -1,114 +1,56 @@
...
+      Syntax: preview/feature/<issue-num>-<description>
+      Example: preview/feature/123-my-preview-feature
+      ```
+    </details>
 
-My change requires unit tests to be written.
-- [ ] Yes
-- [ ] No
-</details>
+- [ ] The **_head(source)_** branch for this pull request is created from a **_preview release_** branch with the correct naming syntax in the following example.
```
Comment (kselena) [inline .github/PULL_REQUEST_TEMPLATE/preview-feature-pr.md:19]: Use colon, not period at the end and use commas in the middle of the sentence.
```suggestion
- [ ] The **_head(source)_** branch for this pull request is a **_preview feature_** branch, with the correct naming syntax, in the following example:
```

File: .github/PULL_REQUEST_TEMPLATE/preview-feature-pr.md [hunk 9]
```diff
@@ -1,114 +1,56 @@
...
+  <details closed><summary>Example</summary>
 
+    ``` xml
+    Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
+    Example: preview/v1.2.3-preview.4
+    ```
+  </details>
 
-<details open><summary>🧪Manual Testing🧪</summary>
+- [ ] The **_base(target)_** branch for this pull request is a **_preview release_** branch with the correct naming syntax in the following example.
```
Comment (kselena) [inline .github/PULL_REQUEST_TEMPLATE/preview-feature-pr.md:28]: Use colon, not period at the end and use commas in the middle of the sentence.
```suggestion
- [ ] The ***_base(target)_** branch for this pull request is a **_preview release_** branch, with the correct naming syntax, in the following example:
```

File: .github/PULL_REQUEST_TEMPLATE/preview-feature-pr.md [hunk 10]
```diff
@@ -1,114 +1,56 @@
...
+
 - [ ] My code follows the coding style of this project.
   - The style is enforced by the *.editorconfig* files in the project and displayed as warnings.  If there is an edge case with a coding style that should be ignored or changed, reach out, and let's discuss it.
+
 - [ ] A ![preview-label](https://user-images.githubusercontent.com/85414302/150838564-33f6044b-55f9-4dd9-8783-1d739de9d92f.png) label has been added to the pull request.
+
+- [ ] I have manually tested my code changes to the best of my ability.
+
 - [ ] All tests passed locally.
   - This is required because unit tests might pass locally but not in the CI environment during the status check process or vise-versa.
```
Comment (kselena) [inline .github/PULL_REQUEST_TEMPLATE/preview-feature-pr.md:55]: Misspelled vice
```suggestion
  - This is required because unit tests might pass locally but not in the CI environment during the status check process or vice-versa.
```

**Review by kselena — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE/preview-feature-pr.md [hunk 11]
```diff
@@ -1,114 +1,56 @@
...
 
+    ``` xml
+    Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
+    Example: preview/v1.2.3-preview.4
+    ```
+  </details>
 
-<details open><summary>🧪Manual Testing🧪</summary>
+- [ ] The **_base(target)_** branch for this pull request is a **_preview release_** branch with the correct naming syntax in the following example.
+  - A **_base(target)_** branch is the branch that is a source branch being merged into it.
```
[the remark this answers is not in the document]
Comment (CalvinWilkinson) [reply .github/PULL_REQUEST_TEMPLATE/preview-feature-pr.md:29]: ❔

File: .github/PULL_REQUEST_TEMPLATE/feature-pr.md (hunk 5 above)
Comment (CalvinWilkinson) [reply .github/PULL_REQUEST_TEMPLATE/feature-pr.md:29]: ❔

File: .github/PULL_REQUEST_TEMPLATE/preview-feature-pr.md [hunk 12]
```diff
@@ -1,114 +1,56 @@
...
-<!-- Go over all of the following points, and put an `x` in all the boxes that apply. -->
-<!-- If you're unsure about any of these, don't hesitate to ask.  We are here to help! -->
-- [ ] Pull request title matches the title of the linked issue.
 - [ ] Associated issue exists and is linked to this pull request.
   - One issue per pull request.
+
+- [ ] The labels attached to this PR match the labels attached to the associated issue.
+
 - [ ] My code follows the coding style of this project.
   - The style is enforced by the *.editorconfig* files in the project and displayed as warnings.  If there is an edge case with a coding style that should be ignored or changed, reach out, and let's discuss it.
```
[the remark this answers is not in the document]
Comment (CalvinWilkinson) [reply .github/PULL_REQUEST_TEMPLATE/preview-feature-pr.md:48]: ❔

**Review by KinsonDigitalAdmin — APPROVED** [review]

**Review by kselena — COMMENTED** [review]
All changes confirmed.

**Review by kselena — APPROVED** [review]
